src/nl/hannahsten/texifyidea/inspections/latex/probablebugs/LatexQuoteInspection.kt

```diff
@@ -0,0 +1,27 @@
+package nl.hannahsten.texifyidea.inspections.latex.probablebugs
```

Comment:
I think this should go under `typesetting` instead of `probable bugs`, because using wrong quotes will compile but produce the wrong typesetting. Perhaps `probablebugs` should have been `compilationissues`, but that doesn't read very nice either.